README.md

```diff
@@ -37,7 +37,7 @@ Before contributing or adding a new feature, please make sure you have already i
 ## 🛠️ Installation Steps
 
 1. Fork the project
-2. Clone the project
+2. Clone the project `git clone https://github.com/EddieHubCommunity/LinkFree.git`
```

Comment:
```suggestion
2. Clone the project by running `git clone https://github.com/EddieHubCommunity/LinkFree.git`
```


__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Inelegant